### ISSUE TITLE
Regarding Udacity Dataset here: http://academictorrents.com/userdetails.php?id=5125

### ISSUE THREAD

**Issue by ujeshmaurya** [cardwing/Codes-for-autonomous-driving-tasks#3]
[Link to dataset](http://academictorrents.com/userdetails.php?id=5125)
We also want to use the dataset from the above link. You told us that you have used datasets 5-8 from the above link, but in the file **2016-10-10.tar.gz** the interpolation file is filled with zeroes(nulls) and in the folder **Ch2_001** the ground truths of testing data is not given.
Please help.

**Comment by cardwing** [comment]
You may also need to download datasets 10 and 11, i.e., Ch2_001: Udacity Self Driving Car and Ch2_002: Udacity Self Driving Car. As to 2016-10-10.tar.gz, if the interpolation file is filled with zeroes(nulls), then just discard this file. You just need to make sure that the training set contains around 404 K frames and the testing set contains 5, 614 frames.

**Comment by ujeshmaurya** [comment]
Datasets 10 is having 0 seeders. Can you suggest any alternate method to download these datasets?

**Comment by cardwing** [comment]
Maybe dataset 14?

**Comment by ujeshmaurya** [comment]
14 is also having 0 seeders.

**Comment by cardwing** [comment]
Try this site: https://github.com/udacity/self-driving-car/tree/master/datasets/CH2.

**Comment by ujeshmaurya** [comment]
[Screenshot](https://drive.google.com/file/d/1U_oW1Ynnj5qYP99VjNryYD6Zq5kYdwRh/view?usp=sharing)
Please see this screenshot, they have deliberately put zeroes in that csv file, because that time the contest was going, can you give us the original (true) **final_example.csv** file, as it is very important for our academic project.

**Comment by cardwing** [comment]
https://github.com/udacity/self-driving-car/blob/master/challenges/challenge-2/CH2_final_evaluation.csv. This is the ground truth label. If you find it is useful, please kindly cite our paper. ^_^
